packages/animation/install

```diff
@@ -17,6 +17,11 @@ if [ $OS_CODENAME == "xenial" ]; then
 fi
 
 # check for supported platform (whitelisted ubuntu versions)
+
+if [ $OS_DISTRIBUTION == "trusty" ]; then
```

Comment:
I think this repository might be out of date. Ubuntu 20.04 is the only supported operating system for new deployments to shinyapps.io at this time.